Course/Course/Presentation/Dates/CourseDatesView.swift

```diff
@@ -35,6 +35,7 @@ public struct CourseDatesView: View {
                     }
                 } else if let courseDates = viewModel.courseDates, !courseDates.courseDateBlocks.isEmpty {
                     CourseDateListView(viewModel: viewModel, courseDates: courseDates)
+                        .padding(.top)
```

Comment:
Please set a specific value for `.padding(.top)`. 
You might use `.padding(.top, 20)` as an example.

Reply:
@volodymyr-chekyrta i have added padding value to this, but i think this should be handled by the parent or maybe we should handle this by adding NavigationTtile to the child view. This same padding/overlapping issue will occur on the handouts tab.